tracer/src/Datadog.Trace/Tags.cs

```diff
@@ -341,6 +341,11 @@ public static partial class Tags
         /// </summary>
         internal const string AwsRegion = "aws.region";
```

Comment:
I'm having trouble figuring out how to handle this. For future integrations, I will not set `aws.region` and `aws.service` in order to remove the tag duplication. But for AWS SQS and SNS, we have to keep them. 

Should I still mark them as obsolete?

Reply:
Added duplication notice as `remarks`, let me know what you think of this. I couldn't mark them as `Obsolete` since the compiler would fail, even after disabling it.

Reply:
I tried doing this workaround, but somehow the compiler wouldn't accept it.
As of now, I'll leave it as a comment, but marking them as obsoletes in the future would be nice!

<img width="958" alt="Screenshot 2023-08-02 at 10 26 37 AM" src="https://github.com/DataDog/dd-trace-dotnet/assets/30836115/968264f7-1a51-41fc-bc3c-3080bbaf16de">

Reply:
🤔 Did you set `error: true`? in the `[Obsolete]` attribute? You can leave that out, and it should work as long as you have those pragmas. 🙂

Reply:
`Tags.cs`
<img width="650" alt="Screenshot 2023-08-02 at 11 20 05 AM" src="https://github.com/DataDog/dd-trace-dotnet/assets/30836115/89df9439-26d7-45b3-b17f-db11dded4211">

`AwsSnsTags.cs` with multiple disable forms
<img width="451" alt="Screenshot 2023-08-02 at 11 20 21 AM" src="https://github.com/DataDog/dd-trace-dotnet/assets/30836115/650ce6d3-08f7-4c5a-8b49-2c8ecdecd288">

<img width="481" alt="Screenshot 2023-08-02 at 11 21 00 AM" src="https://github.com/DataDog/dd-trace-dotnet/assets/30836115/10506111-9927-44a5-8a29-b04b7b14f79d">

Reply:
I tried this and let Rider add the `#pragma` for me and it used `CS0612` (which is the same as `612`)
```
#pragma warning disable CS0612 // Type or member is obsolete
```

Reply:
oh: https://stackoverflow.com/questions/10359117/what-is-the-difference-between-warning-codes-cs0618-and-cs0612

But I'm guessing if you use `[Obsolete(msg, error: true)]` then you can't ignore it because it's an error, not a warning.

Reply:
Found the issue with @lucaspimentel, @andrewlock. I was missing a code change 😞 😥